recipes/abseil/all/conanfile.py

```diff
@@ -49,6 +49,9 @@ def _compilers_minimum_version(self):
             },
         }.get(self._min_cppstd, {})
 
+    def build_requirements(self):
+        self.tool_requires("cmake/3.27.9")
```

Comment:
Let's use version ranges for CMake tool require:

```suggestion
        self.tool_requires("cmake/[>=3.16 <4]")
```